plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -23,6 +23,13 @@
 from ansible.module_utils.common import validation
 from ansible.module_utils.connection import Connection
 
+# Any third party module must be imported as shown. If not ansible sanity tests will fail
+try:
```

Comment:
Can we add an import skip instead similar to this?

https://www.cisco.com/c/dam/en/us/td/docs/dcn/ndfc/1213/articles/ndfc-vpc-fabric/configuring-vpc-fabric-peering.pdf

Reply:
Discussed this and we can keep it as is